app/lib/utils/queryMapper.js

```diff
@@ -27,9 +25,9 @@ function mapServiceType(query) {
   if (query.type) {
     return query.type;
   }
-  if (((query.age) && (query.age === constants.AGE.under16))
-    || ((query.symptoms) && (query.symptoms === constants.SYMPTOMS.yes))) {
-    return constants.SERVICE_TYPES.professional;
+  if (((query.age) && (query.age === constants.age.under16))
+    || ((query.symptoms) && (query.symptoms === constants.symptoms.yes))) {
+    return constants.serviceTypes.professional;
```

Comment:
This would work OK without the brackets around the single values, i.e. `(query.symptoms)`